### PR TITLE
added Opt-Out

### PR DESCRIPTION
**Pull request by nowzig**
added Opt-Out for Google Analytics. 
Tested on Android.

### CODE REVIEW

**Comment by victorsosa** [comment]
THANKS, please update the readme file too.

File: ios/UniversalAnalyticsPlugin.m [hunk 1]
```diff
@@ -100,6 +100,23 @@ - (void) setAnonymizeIp: (CDVInvokedUrlCommand*)command
...
+  CDVPluginResult* pluginResult = nil;
+  NSString* optout = [command.arguments objectAtIndex:0];
+
+  if ( ! _trackerStarted) {
+    pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR messageAsString:@"Tracker not started"];
+    [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
+    return;
+  }
+
+  [[GAI sharedInstance] setOptOut:optout];
```
Comment (victorsosa) [inline ios/UniversalAnalyticsPlugin.m]: Please convert to boolean the pointer, I think you can do it this way:
[[GAI sharedInstance] setOptOut:[optout:boolValue]];

**Comment by nowzig** [comment]
I hope this is fine now

**Comment by victorsosa** [comment]
Yes, thanks; I just need to double check the IOS code

File: ios/UniversalAnalyticsPlugin.m [hunk 2]
```diff
@@ -100,6 +100,23 @@ - (void) setAnonymizeIp: (CDVInvokedUrlCommand*)command
...
+  CDVPluginResult* pluginResult = nil;
+  NSString* optout = [command.arguments objectAtIndex:0];
+
+  if ( ! _trackerStarted) {
+    pluginResult = [CDVPluginResult resultWithStatus:CDVCommandStatus_ERROR messageAsString:@"Tracker not started"];
+    [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
+    return;
+  }
+
+  [[GAI sharedInstance] setOptOut:[optout:boolValue]];
```
Comment (victorsosa) [inline ios/UniversalAnalyticsPlugin.m]: I just tried it and it doesn't compile in IOS, but you can change two line for this:


[[GAI sharedInstance] setOptOut:optout];

File: ios/UniversalAnalyticsPlugin.m [hunk 3]
```diff
@@ -100,6 +100,23 @@ - (void) setAnonymizeIp: (CDVInvokedUrlCommand*)command
   [self.commandDelegate sendPluginResult:pluginResult callbackId:command.callbackId];
 }
 
+- (void) setOptOut: (CDVInvokedUrlCommand*)command
+{
+  CDVPluginResult* pluginResult = nil;
+  NSString* optout = [command.arguments objectAtIndex:0];
```
Comment (victorsosa) [inline ios/UniversalAnalyticsPlugin.m]: bool optout = [[command.arguments objectAtIndex:0] boolValue];

**Comment by victorsosa** [comment]
Thank you, you are awesome

**Pull request merged**